packages/api-explorer/src/JsonForm/configureJsonEditor.js

```diff
@@ -0,0 +1,88 @@
+import it from '../../i18n/it.json'
+
+import antdTheme from "./antd-theme-json-editor";
+import notCustomEditor from "./not-custom-editor";
+import anyOfEditor from "./anyOf-custom-editor";
+import objectsEditor from "./objects-editor";
+
+function setDefaultCustomization (JSONEditor) {
+  const editorsKeys = Object.keys(JSONEditor.defaults.editors)
+
+  editorsKeys.forEach(key => {
+    JSONEditor.defaults.editors[key] = class Customization extends JSONEditor.defaults.editors[key] {
+      postBuild() {
+        super.postBuild()
+    
+        if (this.editjson_control) {
+          this.editjson_control.classList.add('ant-btn-sm')
+          this.editjson_control.style.margin = '0px 8px'
+          this.editjson_control.style.borderRadius = '4px'
+        }
+
+        if (this.addproperty_button) {
+          this.addproperty_button.classList.add('ant-btn-sm')
+          this.addproperty_button.style.margin = '0px 8px'
+          this.addproperty_button.style.borderRadius = '4px'
+        }
+
+        if (this.collapse_control) {
+          this.collapse_control.classList.add('ant-btn-sm')
+          this.collapse_control.style.border = '0px'
+          this.collapse_control.style.background = 'transparent'
+        }
+
+        if (this.toggle_button) {
+          this.toggle_button.classList.add('ant-btn-sm')
+          this.toggle_button.style.border = '0px'
+          this.toggle_button.style.background = 'transparent'
+        }
+       
+        if (this.controls) {
+          this.controls.style.float = 'right'
+          this.controls.style.margin = '5px 0px 0px 10px'
+        }
+        if (this.container !== undefined && this.title !== undefined) {
+          this.container.classList.add('mia-container-wrapper')
+        }
+      }
+    }
+  });
+}
+module.exports = function configureJSONEditor(JSONEditor, intl, setFormSubmissionListener) {
+  JSONEditor.defaults.languages.it = it
```

Comment:
Why it is added and not en?

Reply:
"en" is supported from library itself, there is no need to set them externally